Src/AutoFixture.xUnit.net2.UnitTest/Scenario.cs

```diff
@@ -1,4 +1,5 @@
 ﻿using System;
+using System.ComponentModel.DataAnnotations;
```

Comment:
Yes, the [System.ComponentModel.DataAnnotations](https://www.nuget.org/packages/System.ComponentModel.Annotations/4.1.0) exist in .NET Core, it's just not part of the [NETStandard](https://www.nuget.org/packages/NETStandard.Library/) set of packages so it has to be referenced explicitly – no big deal.